### ISSUE TITLE
Import CSV of presets: 403 Error

### ISSUE THREAD

**Issue by adrianofoschi** [directus/directus#12873]
### Describe the Bug

If you try to import a csv of presets it returns 403 error.

### To Reproduce

- Go to Settings / Presets
- Export csv of your current presets selecting all fields
- Delete all presets
- Import your csv

### Errors Shown

In browser console:

```
{"errors":[{"message":"You don't have permission to access this.","extensions":{"code":"FORBIDDEN"}}]}
```

### What version of Directus are you using?

9.8.0

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres 13

### What browser are you using?

Firefox

### How are you deploying Directus?

docker-compose

**Comment by azrikahar** [comment]
Import endpoint has always denied importing to system collection as seen here:

https://github.com/directus/directus/blob/89aa10b3d7136bff3cd976da15ae80fb9c805179/api/src/services/import-export.ts#L39-L40

However #10978 updated the Presets & Bookmarks layout and subsequently "exposed" the import functionality to the sidebar, thus causing this confusion. 

@rijkvanzanten should this either be hidden in the sidebar for system collections on the App side, or should the import endpoint allow imports to system collections now? (pretty sure this is locked down for a reason, but personally don't have the context on this) 🤔

**Comment by adrianofoschi** [comment]
> 

It's clear now, thank you. I found the bug becouse I am searching a way to backup the presets.
I think they are not included inside the "snapshot" feature.

**Comment by azrikahar** [comment]
> I think they are not included inside the "snapshot" feature.

Yeah snapshot only involves the schema or "shape", if that makes sense. You can refer [this comment](https://github.com/directus/directus/pull/7939#issuecomment-916307666) (and the thread in general) for more info on the reasoning behind it, at least as of now 👍

In the meantime, For the backup of presets or even any other system collections such as data in Project Settings where there is no import/export sidebar to begin with, we can always use the API (eg. https://docs.directus.io/reference/system/presets/) as an alternative to backup & restore data.

**Comment by rijkvanzanten** [comment]
@azrikahar I think we'd want to allow the export/import for admin users only. A lot of the system operations (and the UI for those collections) is admin only already as-is, so that feels like a decent solution 👍🏻
